src/content/get-started/try-a-model/cv-object-detection.mdx

```diff
@@ -69,9 +69,9 @@ The examples below use the SparseZoo stubs. Pass the path to the local model in
 
 ## Inference Pipelines
 
-With the object detection model setup, it can then be passed into a DeepSparse `Pipeline` utilizing the `model_path` argument.
+With the object detection model set up, the model can be passed into a DeepSparse `Pipeline` utilizing the `model_path` argument.
 The SparseZoo stub for the sparse-quantized YOLOv5l model given at the beginning is used in the sample code below.
-It will automatically download the necessary files for the model from the SparseZoo and then compile them on your local machine in the DeepSparse engine.
+It will automatically download the necessary files for the model from the SparseZoo and then compile them on your local machine in the DeepSparse Engine.
```

Comment:
```suggestion
It will automatically download the necessary files for the model from the SparseZoo and then compile them on your local machine in DeepSparse.
```